corehq/form_processor/submission_post.py

```diff
@@ -54,6 +55,15 @@
 )
 
 
+class SubmissionFormContext(object):
+    instance_xml = None
+    supplementary_models = None
```

Comment:
There's no need to define these on the class

```suggestion
```

Reply:
https://github.com/dimagi/commcare-hq/pull/28161/commits/d9cbf8d10561d187d6e152101da86e4cadfdf0a8